src/Uno.UI/UI/Xaml/Controls/MediaPlayerElement/MediaTransportControls.MediaPlayer.cs

```diff
@@ -320,7 +320,7 @@ private void OnPointerEntered(object sender, PointerRoutedEventArgs e)
 		private void OnPointerExited(object sender, PointerRoutedEventArgs e)
 		{
 			_isScrubbing = false;
-			if (_wasPlaying)
+			if (_wasPlaying && _mediaPlayer != null && !_mediaPlayer.PlaybackSession.IsPlaying)
```

Comment:
@rafael-rosa-knowcode What is the intent of this particular change? It would be best to detail why it's there with a comment.

Reply:
@jeromelaban 
his validation was necessary, as it was mistakenly making the video play again when it reached the end and it should have been Paused.
And the Suspended was been call.
In order to successfully finish the tests I had to change that.
But, I can remove it and we include it in another issue. Because it was an error found during the test.

Reply:
Let's include it in another issue, with a test. It feels like it's an edge scenario and I don't get the sequence that can need such an interaction.